test/contracts/front/challenges.js

```diff
@@ -628,24 +674,144 @@ describe('Challenges Contract', () => {
     });
   });
 
-  describe('Calculating cooldown', () => {
-    it('returns 0 if there are less then 4 atlas nodes', async () => {
-      await setNumberOfStakers(2);
-      expect(await getCooldown()).to.equal('0');
+  describe('DMP algorithm testing', () => {
```

Comment:
I would prefer testing library itself. It would allow us to skip nodes onboarding and make tests much much simplier.

Reply:
Fixed
I have moved DMP connected unit tests to the separate file. Nevertheless, some tests are still needed for the round selecting functionality, so I have left them in "Designated shelterer" part.